### DISCUSSION TITLE
Time Tracking - Qual vocês utilizam?

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#1053] [Discussão]
Fala pessoal, queria saber que tipo de ferramentas de time tracking que vocês utilizam, eu utilizava uma extensão do Chrome, agora tenho procurado algo simples para o Trello. 

As vezes acontece de duas pessoas pegarem uma mesma tarefa e acabo me perdendo nos tempos, o ideal pra mim seria um tipo de time tracker dentro de um cartão no Trello. (Se alguém souber me indicar!) haha

**Comment by fabioeilert** [comment]
Utilizei por muito tempo o ever hour(https://app.everhour.com/), é muito bom, tu consegue exportar relatórios depois e ver o que mais te tomou tempo.

**Comment by mahenrique94** [comment]
Dentro do Trello eu nunca usei, mas para monitorar o tempo das minhas tasks eu utilizo o [Toggl](https://toggl.com/), me atende muito bem.

**Comment by espudaro** [comment]
tem o [Harvest](https://www.getharvest.com/), que é de grátis!

**Comment by felipefialho** [comment]
Quando trabalhava remoto usei o [Toggl](https://toggl.com/) por muito tempo e depois conheci o [Harvest](https://www.getharvest.com/), ambos indicados ai em cima.

Os dois são di gratis e funcionam muito bem.

**Comment by thulioph** [comment]
Eu venho utilizando o **Harvest** com esse [power-up pro Trello](https://www.getharvest.com/trello-time-tracking), que faz exatamente o que você precisa @heliomsolivas.

Inclusive nas últimas 3 empresas que eu passei, todas utilizavam o Harvest pelo fato dele gerar um invoice pros clientes, com um detalhamento das horas, e tal, vale a pena dar uma olhada.

**Comment by heliomsolivas** [comment]
Valeu a dica pessoal, coloquei o Harvest aqui pro meu time, parece atender tudo que eu queria! :)

**Comment by RatoX** [comment]
Meu chefe.
